code/modules/paperwork/photocopier.dm

```diff
@@ -419,6 +421,23 @@
 			. = TRUE
 	update_icon()
 
+/obj/machinery/photocopier/proc/ai_text()
+	if(!issilicon(usr))
+		return
+	if(stat & (BROKEN|NOPOWER))
+		return
+	var/text = input("Enter what you want to write:", "Write", null, null) as message
+	if(!text)
+		return
+	if(toner >= 1)
+		playsound(loc, 'sound/goonstation/machines/printer_dotmatrix.ogg', 50, 1)
```

Comment:
```suggestion
		playsound(loc, 'sound/goonstation/machines/printer_dotmatrix.ogg', 50, TRUE)
```

Reply:
it would be weird to change it just here
there is at least 4 `playsound()` in this script that use `1` and `0` instead of `TRUE` and `FALSE`
should I change it everywhere?

Reply:
definitely change it here, if you're feeling up to it, feel free to change the others. We've been transitioning over to using the boolean defines instead of 1s and 0s over the past years.

Reply:
done